common/services/move.js

```diff
@@ -42,6 +42,18 @@ const moveService = {
       })
   },
 
+  getProposed({ createdAtRange = [], fromLocationId } = {}) {
```

Comment:
Would it be easier to pass in the from and to as separate arguments rather than an array?

Reply:
I see your point, but we actually use this pattern elsewhere. I quite like it -- it's semantic, in this instance, and reminiscent of the concept of tuple, that we would use here if we had it in javascript 😃